src/Plugin.php

```diff
@@ -78,4 +81,23 @@ public function wcBootstrap(Site $site, SingleSwitcher $siteSwitcher)
         add_action('woocommerce_new_product', [$wooCommerceGallery, 'saveGalleryIds']);
         add_action('woocommerce_update_product', [$wooCommerceGallery, 'saveGalleryIds']);
     }
+
+    public function acfBootstrap(Site $site, SiteSwitcher $siteSwitcher)
+    {
+        if (!\function_exists('get_field')) {
+            return;
+        }
+
+        // ACF can be included within a theme too - check in after_setup_theme action
+        // https://www.advancedcustomfields.com/resources/including-acf-within-a-plugin-or-theme/
+        \add_action('after_setup_theme', function() use($site, $siteSwitcher) {
+            $store = acf_get_store('values');
+            $this->acfAfterSetupTheme($site, $siteSwitcher, $store);
+        });
+    }
+
+    public function acfAfterSetupTheme(Site $site, SiteSwitcher $siteSwitcher, \ACF_Data $store) {
+        $image = new Image($site, $siteSwitcher, $store);
+        \add_filter('acf/load_value/type=image', array($image, 'acfLoadValue'), 10, 3);
+    }
```

Comment:
@felly sorry to be a bit picky but, make sense that we boot the integration with ACF in one single place to avoid pollutting the Plugin class with a lot of methods in case more integrations will be needed.

Reply:
Thanks @widoz for your support and patience!
This was an attempt to test if `add_filter` is being called with `acf/load_value/type=image`. I will move this code to `acfBootstrap` and remove to corresponding test case.